docs/samples/fluid/README.md

```diff
@@ -0,0 +1,337 @@
+> POST /v1/models/bloom:predict HTTP/1.1
+> Host: fluid-bloom.kserve-fluid-demo.a38c4xxxxxxxxxxxxxxx717177.eu-central-1.elb.amazonaws.com
+> User-Agent: curl/7.86.0
+> Accept: */*
+> Content-Type: application/json
+> Content-Length: 65
+>
+* Mark bundle as not supporting multiuse
+< HTTP/1.1 200 OK
+< content-length: 190
+< content-type: application/json
+< date: Wed, 12 Apr 2023 15:46:16 GMT
+< server: istio-envoy
+< x-envoy-upstream-service-time: 52071
+<
+{
+  "result": "It was a dark and stormy night, and the wind blew with a\nfearful howl. The moon was obscured by a thick cloud, and the stars\nshone dimly. The wind blew the snow from the"
+}
+* Connection #0 to host a38c4xxxxxxxxxxxxxxx717177.eu-central-1.elb.amazonaws.com left intact
+```
+
+### Performance Benchmark
+
+To conduct a performance benchmark, we will compare the scaling time of the inference service with Storage initializer and Fluid across different model artifacts, measuring the time it takes for the service to scale from `0 to 1`.
+
+We will use the following [machine types](https://aws.amazon.com/ec2/instance-types/m5/) for our tests:
+
+* Data: m5.xlarge
+* Workload: m5.2xlarge, m5.4xlarge
+
+We will use the following Fluid runtimes for our tests:
+
+* JindoFS
+
+For the purposes of our benchmark, we will assume that nodes are readily available and that images are cached in the nodes, so we will not be factoring in node provisioning time or image pulling time.
+
+Other assumptions:
+
+* S3 bucket and Cluster are in the same region (eu-central-1)
+* Data are preloaded in workers
+
+
+Command to measure the total time:
+
+```sh
+# Total time includes: pod initialization and running + download model (storage initializer) + load model + inference + network
+curl --connect-timeout 3600 --max-time 3600 -o /dev/null -s -w 'Total: %{time_total}s\n' -H "Content-Type: application/json" -H "Host: ${SERVICE_HOSTNAME}" "http://${INGRESS_HOST}:${INGRESS_PORT}/v1/models/bloom:predict" -d '{"prompt": "It was a dark and stormy night", "result_length": 50}'
+# Total time: 53.037210s, status code: 200
+```
+
+| Model Name                                                                | Model Size (Snapshot) | Machine Type | KServe + Storage Initializer                         | KServe + Fluid(JindoFS)                    |
+|---------------------------------------------------------------------------|-----------------------|--------------|------------------------------------------------------|--------------------------------------------|
+| [ bigscience/bloom-560m ]( https://huggingface.co/bigscience/bloom-560m ) | 3.14GB                | m5.2xlarge   | total: 52.725s (download: 25.091s, load: 3.549s)     | total: 23.286s (load: 4.763s) (2 workers)  |
+| [ bigscience/bloom-7b1 ]( https://huggingface.co/bigscience/bloom-7b1 )   | 26.35GB               | m5.4xlarge   | total: 365.479s (download: 219.844s, load: 102.299s) | total: 53.037s (load: 24.137s) (3 workers) |
```

Comment:
For KServe, it is loading from empty dir to memory. For Fluid it is loading from volume to memory(from my understanding the reading from workers is happening concurrently). Maybe @cheyang can comment on why fluid is faster here.

Reply:
@yuzisun  According to my understand, the empty dir is placed on cloud dis, so it's restricted by the single cloud disk throughput. But for the Fluid, since it has 3 workers, it can provide x3 cloud disk throughput + page cache.  I think it's the reason why fluid is faster than empty dir with larger model load.